src/components/NcDateTime/NcDateTime.vue

```diff
@@ -92,13 +92,13 @@ h4 {
 <template>
 	<span class="nc-datetime"
 		:data-timestamp="timestamp"
-		:title="formattedFullTime"
+		:title="title"
 		v-text="formattedTime" />
 </template>
 
 <script>
-import { computed } from 'vue'
-import { useFormatDateTime } from '../../composables/useFormatDateTime.ts'
+import { computed, toRef } from 'vue'
```

Comment:
That should be missing in Vue 2
```suggestion
import { toRef } from '@vueuse/core'
import { computed } from 'vue'
```

Reply:
Why? `toRef` exists in vue2 as well.
Just the syntax `toRef(() => ...)` does not work (but is not used here)

Reply:
Right, this was backported without getter syntax